src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy

```diff
@@ -401,6 +395,33 @@ class DebugSpec extends BaseSpec {
         assert !response.seatbid
     }
 
+    def "PBS shouldn't populate seatNonBid when debug=#debug, test=#test and requested bidder didn't bid for any reason"() {
```

Comment:
You will also need the same test, but that will check that `returnAllBidStatus` property also works with enabled bebug. Or add a parametrization to `"PBS should populate seatNonBid when returnAllBidStatus=true and requested bidder didn't bid for any reason"` test.